mkt/extensions/views.py

```diff
@@ -190,6 +190,12 @@ class ReviewersExtensionViewSet(CORSMixin, SlugOrIdMixin, MarketplaceView,
     serializer_class = ExtensionSerializer
 
 
+class ReviewerExtensionUpdateViewSet(ReviewerExtensionViewSet):
```

Comment:
I think I would have preferred a custom `@list_route` in the `ReviewerExtensionViewSet` view, and an URL like `/queue/updates/` but not a blocker if you don't agree/don't have time